Code_Exercises/In_Order_Queue/README.md

```diff
@@ -29,6 +30,16 @@ providing the necessary dependencies described in exercise 10 are still met.
 Note that in the USM model when using an in-order `queue` it is no longer
 necessary to chain commands using `event`s.
 
+## Exercise 1
+
+### 1.) Naive In-Order is slow
```

Comment:
```suggestion
### 1.) Naive In-Order Execution
```
I don't think we want to suggest that In-order queues are slow in general. Often they are faster than out of order

Reply:
Agree. Latency-wise, they are better.